src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -74,6 +74,62 @@ public Size(int width, int height)
+        /// <param name="left">Multiplier of type <see cref="int"/>.</param>
+        /// <param name="right">Multiplicand of type <see cref="Size"/>.</param>
+        /// <returns>Product of type <see cref="Size"/>.</returns>
+        public static Size operator *(int left, Size right) => Multiply(right, left);
+
+        /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="left">Multiplicand of type <see cref="Size"/>.</param>
+        /// <param name="right">Multiplier of type <see cref="int"/>.</param>
+        /// <returns>Product of type <see cref="Size"/>.</returns>
+        public static Size operator *(Size left, int right) => Multiply(left, right);
+
+        /// <summary>
+        /// Division operator that divides <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="left">Dividend of type <see cref="Size"/>.</param>
+        /// <param name="right">Divisor of type <see cref="int"/>.</param>
+        /// <returns>Result of type <see cref="Size"/>.</returns>
+        public static Size operator /(Size left, int right)
+        {
+            int invDiv = 1 / right;
+            return new Size(unchecked(left._width * invDiv), unchecked(left._height * invDiv));
+        }
+
+        /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by a <see cref="float"/> producing <see cref="SizeF"/>.
+        /// </summary>
+        /// <param name="left">Multiplier of type <see cref="float"/>.</param>
+        /// <param name="right">Multiplicand of type <see cref="Size"/>.</param>
+        /// <returns>Product of type <see cref="SizeF"/>.</returns>
+        public static SizeF operator *(float left, Size right) => Multiply(right, left);
+
+        /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by a <see cref="float"/> producing <see cref="SizeF"/>.
+        /// </summary>
+        /// <param name="left">Multiplicand of type <see cref="Size"/>.</param>
+        /// <param name="right">Multiplier of type <see cref="float"/>.</param>
+        /// <returns>Product of type <see cref="SizeF"/>.</returns>
+        public static SizeF operator *(Size left, float right) => Multiply(left, right);
+
+        /// <summary>
+        /// Division operator that divides <see cref="Size"/> by a <see cref="float"/> producing <see cref="SizeF"/>.
+        /// </summary>
+        /// <param name="left">Dividend of type <see cref="Size"/>.</param>
+        /// <param name="right">Divisor of type <see cref="int"/>.</param>
+        /// <returns>Result of type <see cref="SizeF"/>.</returns>
+        public static SizeF operator /(Size left, float right)
+        {
+            float invDiv = 1.0f / right;
+            return new SizeF(left._width / invDiv, left._height / invDiv);
```

Comment:
The test case with input [Int.Max/Min, Float.Max/Min] (4 combinations) failed as soon as I corrected / to *. But I'm surprised why all of them passed when it was /....